docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -487,22 +496,67 @@ export default function AppLayoutDocsFooter(props) {
                   {t('submit')}
                 </Button>
               </DialogActions>
-              {rating !== 1 && (
-                <Typography id="feedback-description" color="text.secondary">
-                  {t('feedbackMessageToGitHub.usecases')}
-                  <br />
-                  {t('feedbackMessageToGitHub.callToAction.text')}
+            </Box>
+          </form>
+        </Collapse>
+        <Divider sx={{ mb: 4 }} />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', md: 'row' },
+            justifyContent: 'space-between',
+            gap: { xs: 4, lg: 12 },
+          }}
+        >
+          <Box flexGrow={1}>
+            <Typography variant="body2" fontWeight="bold">
+              Keep up to date
+            </Typography>
+            <Typography variant="body2" color="text.secondary" sx={{ mb: 2 }}>
+              Join our newsletter for regular updates. No spam ever.
+            </Typography>
+            <EmailSubscribe fullWidth sx={{ mb: 1 }} />
+          </Box>
+          <Divider sx={{ display: { xs: 'block', sm: 'none' } }} />
+          <Stack direction="column" spacing={3}>
+            <Typography variant="body2" fontWeight="bold">
+              Engage further with us
+            </Typography>
+            <Stack direction="column" spacing={1}>
+              <Stack direction="row" alignItems="center" spacing={1}>
+                <BugReportRoundedIcon fontSize="small" color="disabled" />
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 0 }}>
+                  Need answers?
                   <Link
                     href={`${process.env.SOURCE_CODE_REPO}/issues/new?template=${process.env.GITHUB_TEMPLATE_DOCS_FEEDBACK}&page-url=${window.location.href}`}
                     target="_blank"
+                    sx={{ ml: 0.5 }}
                   >
                     {t('feedbackMessageToGitHub.callToAction.link')}
                   </Link>
                 </Typography>
-              )}
-            </Box>
-          </form>
-        </Collapse>
+              </Stack>
+              <Stack direction="row" alignItems="center" spacing={1}>
+                <ChangeCircleRoundedIcon fontSize="small" color="disabled" />
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 0 }}>
+                  Past releases?
+                  <Link href={ROUTES.roadmap} target="_blank" sx={{ ml: 0.5 }}>
```

Comment:
Wrong link

Reply:
I'll resolve this comment as there's an over-arching conversation about this happening already 👍